modules/importing-repositories/procedure-bulk-import-scaffolder-template.adoc

```diff
@@ -0,0 +1,66 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="procedure-bulk-import-scaffolder-template"]
+= Understanding Bulk Import Scaffolder template input parameters
+
+You can use the Bulk Import plugin to run a Scaffolder template task with specified parameters. As an administrator, you must use these parameters within the template.
+
+The Bulk Import plugin analyzes Git repository information and provides the following parameters for the Scaffolder template task:
+
+`repoUrl`:: Normalized repository URL in the following format:
++
+[source,yaml]
+----
+  ${gitProviderHost}?owner=${owner}&repo=${repository-name}
+----
++
+For example, when you run the process, the plugin transforms `https://github.com/redhat-developer/rhdh-plugins` into `github.com?owner=redhat-developer&repo=rhdh-plugins`.
+
+`name`:: The repository name.
++
+For example, for `https://github.com/redhat-developer/rhdh-plugins`, the `name` is `rhdh-plugins`.
+
+`organization`:: The repository owner, which can be a user nickname or organization name.
++
+For example, for `https://github.com/redhat-developer/rhdh-plugins`, the `organization` is `redhat-developer`.
+
+`branchName`:: The proposed repository branch. By default, the proposed repository branch is `bulk-import-catalog-entity`.
+
+`targetBranchName`:: The default branch of the Git repository.
+
+`gitProviderHost`:: The Git provider host parsed from the repository URL.
++
+For example, for `https://github.com/redhat-developer/rhdh-plugins`, the `gitProviderHost` is `github.com`. You can use this parameter to write `Git-provider-agnostic` templates.
+
+To use these parameters in a Scaffolder template:
```

Comment:
is this a hidden procedure? the rest of the module reads more like a concept module but the module type is specified as procedure